backend/core/src/sms/services/sms.service.ts

```diff
@@ -7,18 +9,23 @@ import { TwilioService } from "./twilio.service"
 
 @Injectable({ scope: Scope.REQUEST })
 export class SmsService {
-  constructor(private readonly twilio: TwilioService) {}
+  constructor(private readonly twilio: TwilioService, private readonly userService: UserService) {}
+
+  async sendNewListingNotification(listing: Listing): Promise<StatusDto> {
+    // TODO(https://github.com/CityOfDetroit/bloom/issues/705): when Detroit Home Connect has a
+    // URL, update this message so that it includes a link to the new listing.
+    const notificationBody = `A new listing was recently added to Detroit Home Connect: ${listing.name}.`
```

Comment:
Might also want to leave a TODO for potentially translating this string.

Reply:
Oh good call!